torchrl/modules/tensordict_module/exploration.py

```diff
@@ -17,13 +22,165 @@
 
 __all__ = [
     "EGreedyWrapper",
+    "EGreedyModule",
     "AdditiveGaussianWrapper",
     "OrnsteinUhlenbeckProcessWrapper",
 ]
 
 
+class EGreedyModule(TensorDictModuleBase):
+    """Epsilon-Greedy module.
+
+    This module updates the action in a tensordict to an epsilon greedy one.
+
+    Keyword Args:
+        eps_init (scalar, optional): initial epsilon value.
+            default: 1.0
+        eps_end (scalar, optional): final epsilon value.
+            default: 0.1
+        annealing_num_steps (int, optional): number of steps it will take for epsilon to reach the eps_end value
+        action_key (NestedKey, optional): if the policy module has more than one output key,
+            its output spec will be of type CompositeSpec. One needs to know where to
```

Comment:
```suggestion
            its output spec will be of type :class:`torchrl.data.CompositeSpec`. One needs to know where to
```

Reply:
What does this mean?
> One needs to know where to find the action spec

Reply:
tbf all this is from the previous module, i didn't change it

Reply:
i will now